diff --git a/commerce_reports.module b/commerce_reports.module
index 1ed99d0..626c40e 100755
--- a/commerce_reports.module
+++ b/commerce_reports.module
@@ -75,9 +75,9 @@ function commerce_reports_block_info() {
     'commerce_reports_customers' => array(
       'info' => t('Commerce Reports: Customer statistics'),
     ),
-    'commerce_reports_customer_map' => array(
-      'info' => t('Commerce Reports: Customer map'),
-    ),
+//    'commerce_reports_customer_map' => array(
+//     'info' => t('Commerce Reports: Customer map'),
+//    ),
   );
 }
 
@@ -94,8 +94,8 @@ function commerce_reports_block_view($delta = '') {
       return commerce_reports_sales();
     case 'commerce_reports_customers':
       return commerce_reports_customers();
-    case 'commerce_reports_customer_map':
-      return commerce_reports_customer_map();
+//    case 'commerce_reports_customer_map':
+//      return commerce_reports_customer_map();
   }
 }
 
